### DISCUSSION TITLE
遇到不熟的熟人不打招呼也不会觉得尴尬 | /posts/politeness/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#593] [Announcements]
https://yuanfan.rbind.io/posts/politeness/

**Comment by HushWay** [comment]
> “我没有罪，我不可耻，不可悲，不可怜。”

通过留言发现了你的博客，羡慕你的性格与思考力。读到这句的时候觉得心头一热，我也总是陷入无尽的自卑与自责中，我明白接纳自我很重要，可是不在意他人的目光真的好难

**Reply by earfanfan** [reply]
接纳自我和不在意别人的目光并不是互相冲突的，我也觉得这两者都很难，不过好在好好活着就有希望。慢慢来嘛，先对现在的自己宽容一点。

**Reply by HushWay** [reply]
谢谢你，年前感情失意，痛苦与自我怀疑反复煎熬，这也是为什么我看到你这句话会被温暖到的原因。虽然没完全走出来，我明白面对就有希望，好好活着就有希望。

**Reply by earfanfan** [reply]
原来你是遇到了这个人生课题啊，我上一次面对这个考验是好多年前，记得当时通过考验的进度条也是容易反复，那种努力面对了好久一下子回退到解放前的感觉确实很糟糕，不过后来时间这个包治百病的庸医真得治好了我，进度条最终走到了百分百。关于“接纳自我”，我现在是这样看待的，大约就是无论如何，“我”在这个世界这个宇宙上一定是有一个位置，只要我愿意就能存在。而“他人的目光”需要分辨着看，如果看到我们的人是带着恶意的，这种目光尽量不去在意，这样也是保护自己，如果看到我们的人对待我们的态度是平等和尊重，那么这种目光可以在意，但也不一定就对。一方面，我的存在就是存在，另一方面，如果我要认清自己在这个世界上是什么样子，恐怕还是要能够看到他人的存在的。

**Reply by HushWay** [reply]
你说的很有道理，应该辩证地去看待别人的看法，从善如流。正是分离过程中坦诚的交流让我看清自己原本的、真正的样子。如此说来，客观的目光是一面镜子，如实观照方能自知
